### DISCUSSION TITLE
T does not respond.

### DISCUSSION THREAD

**Discussion by saranick** [xournalpp/xournalpp#3377] [General]
T does not respond.
I try to type text but it does not write.
On the same computer I had the older version of the xurnal and I had no problem.
The computer is running ubuntu 20.04

**Comment by personalizedrefrigerator** [comment]
How did you install Xournal++ (AppStore/snap, Flathub, AppImage,...)?

Can you type text into any of the input boxes in the settings dialog?

Are you currently using v1.1.0? What was the old version of Xournal++?

If you save a file with text from the old version of Xournal++ and open it with the new version, can you edit the existing text (by clicking on it with the T tool?)

Do you experience the same behavior with the AppImage, if you weren't using it already?
<details><summary>Using the AppImage</summary>

 * Download the [AppImage](https://github.com/xournalpp/xournalpp/releases/download/1.1.0/xournalpp-1.1.0-x86_64.AppImage) from the releases page (it's under "Assets" and the releases page is linked to on the main page of this GitHub repository).

 * Mark it as executable (in Files: Right-click, allow executing the file as a program. In a terminal: <code>chmod u+x ~/Downloads/xournalpp-1.1.0-x86_64.AppImage</code>.)

 * Run it (In Files: Right click > Run. In a terminal: <code>cd ~/Downloads && ./xournalpp-1.1.0-x86_64.AppImage</code> if you saved it in Downloads).

</details>

**Reply by saranick** [reply]
Good Evening.
Τhanks for the prompt response.
I use it for about 1 year, so I do not remember how I installed it.
looking of course in the ubuntu software I see that version 1.1.0 is installed and has been updated on September 7.
Yes I can write text in input boxes in the settings dialogs.
Because the upgrade was done automatically I no longer have access to the old version of xournal++
I have not used appimage.
Because my English is not very good, (I use google translate) you suggest me to install appimage?

**Reply by personalizedrefrigerator** [reply]
You <i>might</i> not experience the issue while using the `AppImage`.

I will try installing `Xournal++` from the App Store on Ubuntu 20.04 and see if the text tool works for me. If it doesn't, then there is a bug in the App Store (`Snap Store`) distribution of `Xournal++`!

## What I think could be causing the issue

I think one of these might be causing the issue:
 * There might be an issue with the version of the `GTK` library used by your copy of `Xournal++`. `Xournal++` uses `GTK` to handle input, create windows, and show output. The `AppImage` might use a different version of some libraries than the App Store version of `Xournal++`.
 * Settings from an old version of `Xournal++` might be breaking the text tool. If this is the case, installing the `AppImage` shouldn't fix the issue. [**Edit:** This seems most likely, see reply below. ]
 * **Edit:** It might also be related to your keyboard layout or input method. See #3356, #

If the issue is being caused by a setting, you can try deleting settings-related files.

## To reset all settings:
 1. In a terminal, run `mv ~/.xournalpp ~/.xournalpp_backup`. This renames the folder `.xournalpp` to `.xournalpp_backup`. Older versions of `Xournal++` used `~/.xournalpp` to store settings. If version 1.1.0 can't find its settings, it looks in `~/.xournalpp` and copies those settings to `~/.config/xournalpp`.
 2. Also in a terminal, run `mv ~/.config/xournalpp ~/.config/xournalpp_backup`. This renames the folder used by version 1.1.0 to `xournalpp_backup`. If both `~/.config/xournalpp` and `~/.xournalpp` are missing, version 1.1.0 will reset all settings to their defaults. **Edit:** If `~/.config/com.github.xournalpp` exists, also re-name it.

<sub>**Edit:** Fixed mistake in explanation of how to reset settings.</sub>

**Reply by personalizedrefrigerator** [reply]
Hi,
I've installed `Xournal++ v1.1.0` from the App Store on Ubuntu 20.04. The text (T) tool works for me.

<details>

https://user-images.githubusercontent.com/46334387/132556761-27ebc9d1-17d4-40b1-9ddf-be6dab89eb4a.mp4

</details>


----

As such, I suggest you reset `Xournal++`'s settings (by renaming or deleting the following folders: `~/.xournalpp` and `~/.config/xournalpp`).

If the above doesn't work, you may wish to uninstall and reinstall Xournal++.

----
**Note:** If you prefer to communicate in a language other than English, `GitHub` has a new &ldquo;Translate to English&rdquo; feature I can try to use. It might be easier for me to use this than for you to use Google Translate :).

**Comment by personalizedrefrigerator** [comment]
See https://github.com/xournalpp/xournalpp/issues/3277 and https://github.com/xournalpp/xournalpp/issues/3203. As described there, you may need to set one of your devices to "Mouse + Keyboard Combo" in preferences.

**Comment by personalizedrefrigerator** [comment]
Also see #3389
